packages/block-library/src/query/edit/query-toolbar.js

```diff
@@ -39,7 +40,7 @@ export default function QueryToolbar( {
 	return (
 		<>
 			{ ! query.inherit && (
-				<ToolbarGroup>
+				<BlockControls group="block">
```

Comment:
Thanks @ntsekouras, I can't believe I missed that 👍 

I'm starting to think the settings dropdown might fit better in the meta/default section rather than in the block one, as it defines the query rather than the visual presentation. What do you think?

Reply:
No strong opinions here as this control will probably move to inspector controls. There is an issue about the overall grouping/positioning all this block's controls, so changes will happen relatively soon.

Reply:
Reverted the changes. Adding the group attribute to the existing `BlockControl` would still require adapting the `QueryToolbar` internally as it uses `ToolbarGroup` and `Dropdown`, so I'm leaving it for a later PR so that we can ship this one.